qiskit/extensions/unitary.py

```diff
@@ -191,22 +191,13 @@ def qasm(self):
             _qasm_escape_gate_name(self.label) if self.label else "unitary" + str(id(self))
         )
 
-        # map from gates in the definition to params in the method
-        bit_to_qasm = OrderedDict()
```

Comment:
The OrderedDict() class is no longer being used, it should be better to remove that import to avoid any import issues.

Reply:
This is giving an error in further tests done using tox -elint due to it being an unused-import and can cause problems for future developers when running tests.

Reply:
Thanks, this was a good catch. I'm not sure how this slipped through CI here (I relied a bit too heavily on it in my code review earlier) but this also just started failing on CI today, the fix for this is here: https://github.com/Qiskit/qiskit-terra/pull/8366